packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -52,6 +52,7 @@ import {
 import {StatsAnalyzer, EVENTS as StatsAnalyzerEvents} from '../statsAnalyzer';
 import NetworkQualityMonitor from '../networkQualityMonitor';
 import LoggerProxy from '../common/logs/logger-proxy';
+import EventsUtil from '../common/events/util';
 import Trigger from '../common/events/trigger-proxy';
```

Comment:
Won't we also remove this import here or is still getting used in this file? If it is still being used, we are not planning to remove it from other places?

Reply:
No, @Kesari3008. We won't remove it from the file because it's being used by other methods. 

The reason why we got rid of `TriggerProxy` inside the `plugin-meetings` for Voicea alone is that, TriggerProxy logs the payload and for Voicea while doing so, it lets out PII information. To avoid that, we log separately and emit event from inside iteslf. While logging, we don't log the payload for any of the Voicea events.